
bug description: boot a guest, open VNC console, and when it's still in BIOS / bootloader sequence, type as many keys as you can, move your mouse as much as you can. If needed, just reboot the guest.

Linux guest will boot with "i8042.c: No controller found" and no keyboard.

Further discussion seem to lead to a conclusion that there is some obscure bug in Qemu BIOS which makes this problem visible.

Index: kvm-86/hw/ps2.c
===================================================================
--- kvm-86.orig/hw/ps2.c	2009-06-15 13:31:24.000000000 +0200
+++ kvm-86/hw/ps2.c	2009-06-15 13:31:50.000000000 +0200
@@ -70,7 +70,7 @@
 #define MOUSE_STATUS_ENABLED    0x20
 #define MOUSE_STATUS_SCALE21    0x10
 
-#define PS2_QUEUE_SIZE 256
+#define PS2_QUEUE_SIZE 15
 
 typedef struct {
     uint8_t data[PS2_QUEUE_SIZE];
